docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '3'
```

Comment:
We could DRY this up by moving it to terra-toolkit and curl'ing in the travis.yml file before running the docker-compose commands.  We could have the image name be an environment variable.  We also need to have a tag.  I'm thinking it could be the travis build id or travis build number if the branch is not master and "latest" otherwise.  Thoughts?

Reply:
However, should the image name be an environment variable? Didn't we want to lock that version down?